src/synthesizer/grid.py

```diff
@@ -303,6 +303,20 @@ def _get_spectra_grid(self, spectra_to_read):
             for spectra_id in self.available_spectra:
                 self.spectra[spectra_id] = hf["spectra"][spectra_id][:]
 
+            # THIS IS A HACK BECAUSE SW CHANGED THE AGN EMISSIONS WITHOUT
+            # UPDATING THE TEST GRIDS AND ONLY HE HAS ACCESS TO THE DROPBOX.
+            # THIS WILL BE REMOVED WHEN THE GRIDS ARE UPDATED.
+            if "nlr" in self.grid_filename or "blr" in self.grid_filename:
```

Comment:
I understand this is in flux, but did we agree upon to have specifically add nlr or blr to the AGN grid names? I assumed it was simply an AGN grid with a wide range of hydrogen densities and the user gets to pick sensible densities for the two regions?